app/src/main/java/org/p2p/wallet/utils/DateTimeUtils.kt

```diff
@@ -156,6 +156,10 @@ object DateTimeUtils {
         return String.format(Locale.US, "%02d:%02d", m, s)
     }
 
+    fun checkTimerIsOver(timeMs: Long, expireDurationMs: Long): Boolean {
```

Comment:
well, I don't think DateTimeUtils should know about any timer staff :)) 
maybe `isTimestampExpired`

Reply:
that's what I wanted! - isTimestampExpired